### ISSUE TITLE
Injection Temperature accordion box is too wide.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#223]
@arouinfar noted that the Injection Temperature accordion box is a bit wider than the other accordion boxes above it:

<img width="308" alt="screenshot_3245" src="https://github.com/phetsims/gas-properties/assets/3046552/c66dbe00-4a71-4e1a-9bd6-30116534e4b8">

**Comment by pixelzoom** [comment]
This sim is using a very old workaround for making panels have a fixed width, and making a group of panels all have the same width.  See `fixedWidth?: number`.  When I worked on dynamic layout, I decided that I wanted to avoid touching `fixedWidth`.  But this issue and layout issue in #210 have convinced me that I should convert to scenery layout patterns for handling this.  

So this issue will involve more than just fixing the width of the Injection Temperature panel.  It's will include updates to layout for all Panels and AccordionBoxes.

**Comment by pixelzoom** [comment]
The width problem with InjectionTemperatureAccordionBox was introduced between [1.1.0-dev.4](https://phet-dev.colorado.edu/html/gas-properties/1.1.0-dev.4/phet/gas-properties_en_phet.html) (2022-11-17) and [1.1.0-dev.5](https://phet-dev.colorado.edu/html/gas-properties/1.1.0-dev.5/phet/gas-properties_en_phet.html) (2023-05-22).

**Comment by pixelzoom** [comment]
Injection Temperature accordion box is fixed, and (more importantly) layout of panels and accordion boxes has been updated to scenery layout.  For each set of panels/boxes that have the same fixed width, the approach is to wrap them in a VBox with these options:

```ts
      stretch: true,
      minContentWidth: PANELS_WIDTH,
      maxWidth: PANELS_WIDTH
```

@arouinfar Note that I also reduced that font size of titles from 18 to 16, as we discussed last week.  This gives us a little more space for historgrams, and still looks great. 

Closing.
